web/beacon-app/src/features/onboarding/components/steps/workspace/form.tsx

```diff
@@ -0,0 +1,116 @@
+import { t } from '@lingui/macro';
+import { ErrorMessage, Form, FormikHelpers, FormikProvider } from 'formik';
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+
+import { stringify_org } from '@/utils/slugifyDomain';
+
+import { useNewWorkspaceForm } from '../../../useNewWorkspaceForm';
+import StepButtons from '../../StepButtons';
+
+const DOMAIN_BASE = 'https://rotational.io/';
```

Comment:
Currently, the `DOMAIN_BASE` is `https://rotational.app`. Could you confirm which domain should be used here?